push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -111,6 +111,9 @@ private void updateStatusOfPushMessageInformation(final PushMessageInformation p
         variantMetricInformation.setReceivers(receivers);
         variantMetricInformation.setDeliveryStatus(deliveryStatus);
         variantMetricInformation.setReason(reason);
+
+        pushMessageInformation.getVariantInformations().add(variantMetricInformation);
```

Comment:
This is a reason why I got 2 receivers.. 

The variant information is added to PushMessageInformation in the `MetricCollector#updateExistingMetric`: https://github.com/matzew/aerogear-unifiedpush-server/blob/AGPUSH-1374/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/MetricsCollector.java#L66

Since it is already added in the `NotificationDispatcher`, it re-adds "existing" variant metric.

By removing this line it starts to work correct.